src/test/java/org/apache/cassandra/sidecar/routes/CassandraSettingsServiceTest.java

```diff
@@ -0,0 +1,118 @@
+package org.apache.cassandra.sidecar.routes;
```

Comment:
can we add the license header to this file as well

Reply:
added license information.